demo/full/scripts/controllers/ControlBar.jsx

```diff
@@ -48,7 +55,19 @@ function ControlBar({
           disabled={isStopped}
         />
         { positionElement }
+        { (isContentLoaded && isNearLiveEdge) ? <div className="dot"></div> : null}
         <div className="controls-right-side">
+          { isLive ?
+            <Button
+              className={"control-live" + (hasSeeked ? " activated" : "")}
```

Comment:
What if we're not in a low-latency contents but we have bufferized for a long time? Are we now stuck?

Why not just enabling it based on the current distance with the live again?